libs/gnutls/Makefile

```diff
@@ -178,16 +183,20 @@ ifeq ($(CONFIG_GNUTLS_CRYPTODEV),y)
 CONFIGURE_ARGS += --enable-cryptodev
 endif
 
```

Comment:
You can make this a one liner. $( if (x,,). This can keep it part of the original CONFIGURE_ARGS.